code/datums/emergency_calls/pred_hunt/hunting_calls.dm

```diff
@@ -172,3 +173,67 @@
 	hostility = TRUE
 	max_xeno_t3 = 3
 	max_xeno_t2 = 3
+
+/datum/emergency_call/young_bloods //YOUNG BLOOD ERT ONLY FOR HUNTING GROUNDS IF SOME MOD USES THIS INSIDE THE MAIN GAME THE COUNCIL WONT BE HAPPY (Joe Lampost)
+	name = "Template"
+	var/blooding_name
+	probability = 0
+	name_of_spawn = /obj/effect/landmark/ert_spawns/distress/hunt_spawner/pred
+	shuttle_id = ""
+
+/datum/emergency_call/young_bloods/remove_nonqualifiers(list/datum/mind/candidates_list)
+	var/list/datum/mind/youngblood_candidates_clean = list()
+	for(var/datum/mind/youngblood_candidate in candidates_list)
+		if(check_timelock(youngblood_candidate.current?.client, JOB_SQUAD_ROLES_LIST, time_required_for_youngblood) && (youngblood_candidate.current?.client.get_total_xeno_playtime() >= 40 HOURS))
+			youngblood_candidates_clean.Add(youngblood_candidate)
+			continue
+		if(youngblood_candidate.current)
+			to_chat(youngblood_candidate.current, SPAN_WARNING("You didn't qualify for the ERT beacon because you did not meet the required hours for this role (40 hours as a Marine and Xeno) or you are already whitelisted as a Predator"))
+		if(youngblood_candidate.current?.client?.check_whitelist_status(WHITELIST_YAUTJA) || jobban_isbanned(youngblood_candidate.current?.client, ERT_JOB_YOUNGBLOOD))
+			continue
```

Comment:
You should move the WL and Jban check to be the first thing on the for loop, otherwise it won't do anything